luten-scheduling/src/instances/random.rs

```diff
@@ -0,0 +1,202 @@
+use rand::{Rng, thread_rng};
+use rand::distributions::{IndependentSample, Sample};
+use rand::distributions::normal::Normal;
+use rand::distributions::range::Range;
+use std::collections::HashMap;
+
+use types::*;
+use WorkDay::*;
+use util::clamp;
+
+pub struct RatingDistribution<T, U, V> where
+    T: IndependentSample<f64>,
+    U: IndependentSample<f64>,
+    V: IndependentSample<f64>,
+{
+    pub available_blocks_per_day: u16,
+    // distribution used to sample overall number of slots rated as `Good` or `Tolerable`
+    pub rated_slots: T,
+    // proportion of slots rated good vs tolerable
+    pub good_slot_percentage: U,
+    // percentage of slots rated together as a block of four slots
```

Comment:
These three comments should be doc comments: those start with three slashes and an uppercase character.